docs/_source/docs/stack_config.rst

```diff
@@ -24,8 +24,11 @@ particular Stack. The available keys are listed below.
 -  `parameters`_ *(optional)*
 -  `protected`_ *(optional)*
 -  `role_arn`_ *(optional)*
+-  `cloudformation_service_role`_ *(optional)*
 -  `iam_role`_ *(optional)*
```

Comment:
would it make sense to mark these as deprecated here as well?  something like `[Deprecated] iam_role_ *(optional)*`

Reply:
I was thinking about that... that's easy enough to do.